zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneController.java

```diff
@@ -59,16 +61,18 @@ public ResponseEntity<OrchestratorZoneResponse> getZone(@NotBlank(message = MAND
 
     @PostMapping
     @Transactional
-    public ResponseEntity<?> createOrchestratorZone(@RequestBody @Valid OrchestratorZoneRequest orchestratorZoneRequest )
-        throws OrchestratorZoneServiceException, IOException {
+    public ResponseEntity<OrchestratorZoneResponse> createOrchestratorZone(
+        @RequestBody @Valid OrchestratorZoneRequest orchestratorZoneRequest) {
         zoneService.createZone(orchestratorZoneRequest);
-        return ResponseEntity.accepted().build();
+        OrchestratorZoneResponse body =
```

Comment:
In getZone(), OrchestratorZoneResponse is returned by the zoneService.getZoneDetails().  In createOrchestratorZone(), OrchestratorZoneResponse is built in the controller. For consistency we should use one or the other.

Reply:
Sure I will move the code to inside the service